libbpf-tools/trace_helpers.h

```diff
@@ -20,6 +20,25 @@ const struct ksym *ksyms__map_addr(const struct ksyms *ksyms,
 const struct ksym *ksyms__get_symbol(const struct ksyms *ksyms,
 				     const char *name);
 
+struct sym {
+	const char *name;
+	unsigned long start;
+	unsigned long size;
+};
+
+struct syms;
+
+struct syms *syms__load(int tgid);
```

Comment:
as Dave mentioned, you probably would want to have "syms__load_pid" and "syms_load_file" variants

Reply:
Yes, changed to these two variants, thanks!